docs: add manage dashboards as code overview (#5448) (#7029)
elastic/docs-content: explore-analyze/dashboards/building.md

| [Create dashboards programmatically](create-dashboards-programmatically.md) | Automating deployments, managing dashboards and visualizations as code, CI/CD pipelines | Saved dashboard or visualization |
| [Create dashboards using AI](create-dashboards-using-ai.md) | Generating dashboards from natural language, or building AI tools that create dashboards | Dashboard through chat that you save when ready, or saved dashboard using the API |

Once you have a dashboard, you can enhance it:

* [Add filter controls](add-controls.md) to let viewers explore the data interactively
* [Add drilldowns](drilldowns.md) to navigate between dashboards or to external URLs

> * [Arrange and resize panels](arrange-panels.md) to get the layout you want

Author: ```suggestion
* [Arrange and resize panels](arrange-panels.md) to optimize its layout
```
